Gruntfile.js

```diff
@@ -8,27 +8,15 @@ module.exports = function (grunt) {
 
   // Node and client side JS have slightly different JSHint directives
   // We'll create 2 versions with .jshintrc as a baseline
-  var browserJSHint = grunt.file.readJSON('.jshintrc');
-  var nodeJSHint = grunt.file.readJSON('.jshintrc');
-
-  // Don't throw errors for expected Node globals
-  nodeJSHint.node = true;
+  var JSHint = grunt.file.readJSON('node_modules/mofo-style/linters/.jshintrc');
```

Comment:
that feels like a thing worth filing against grunt-contrib-jshint, afterwhich we can go with the JSON.parse approach while they mull over whether or not to allow option extensions after loading the profile from file.